pkgs/tools/X11/imwheel/default.nix

```diff
@@ -10,10 +10,7 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libX11 libXext libXi libXmu libXt libXtst ];
 
-  postPatch = ''
-    substituteInPlace Makefile.in --replace "ETCDIR = " "ETCDIR = $out"
-    substituteInPlace util.c --replace "/etc/X11/imwheel" "$out/etc/X11/imwheel"
-  '';
+  installFlags = ["sysconfdir=/etc" "ETCDIR=/etc" ];
```

Comment:
nit: inconsistent spacing around brackets

Reply:
I believe you should do this
```
  makeFlags = [
    "sysconfdir=/etc"
    "ETCDIR=/etc"
  ];

  installFlags = [
    "sysconfdir=${placeholder "out"}/etc"
    "ETCDIR=${placeholder "out"}/etc"
  ];
```

So when things are compiled, I hope they get that value because of the `makeFlags`.
But for installation they still end up in `$out`. With this it fails with a permission denied error, which is weird because it really should fail the build entirely (since a step in the phase failed...).

Reply:
Also, I checked the source for that patched entry and it's
```
	char fname[2][1024]={"","/etc/X11/imwheel/imwheelrc"}, line[1024], *p, *q, winid[1024];
```

So it's already hardcoded, so disregard my point about "compiled" as there's no substitution going on here.
However this is still needed so it can read the config from the dir `/etc/X11/imwheel/imwheelrc`